lib/stats-writer-plugin.js

```diff
@@ -6,6 +6,8 @@ const path = require("path");
 const INDENT = 2;
 const DEFAULT_TRANSFORM = (data) => JSON.stringify(data, null, INDENT);
 
+let IS_FIRST_START_DEVSERVER = true;
```

Comment:
maybe, `IS_FIRST_RUN_WO_HMR`?

Reply:
I think that `IS_FIRST_START_DEVSERVER` is okay because it's not only for `HMR`